cmd/podman/mount.go

```diff
@@ -80,35 +85,39 @@ func mountCmd(c *cli.Context) error {
 		}
 	}
 
+	if c.Bool("all") && c.Bool("latest") {
+		return errors.Errorf("--all and --latest cannot be used together")
+	}
+
```

Comment:
I see you are using checkAllAndLatest(c) in `umount.go`, any reasons to not also use it here?

Reply:
podman mount is allowed to have no containers.  And checkAllAndLatest returns an error if the user did not specify --all, or --latest, or containers.

podman mount 
will list all mounted containers.